### DISCUSSION TITLE
MaxLFQ for Deuterium Enriched Samples

### DISCUSSION THREAD

**Discussion by aDenos24** [Nesvilab/FragPipe-Analyst-doc#69] [Q&A]
Since switching from PEAKS to Fragpipe, a question about normalization has emerged. We had an 'offline' normalization we performed on peaks protein exports but with fragpipe I have been using the MaxLFQ results directly. I understand normalization strategies can enrich data in different ways, but when I compared the results of both analysis types on the same data, I got a correlation coefficient of 0.2. I am wondering if perhaps MaxLFQ is not the best for my type of data (Question 1). I have samples that are deuterium enriched at timepoints 0-2 days. I am wondering if the normalization of "intensity" is enough, or if from that output I should use the offline normalization I was performing before (this does a log2 of the data, normalizes the means, and then the slopes). I want to be careful to not over normalize my data (Question 2). Any suggestions?

**Comment by fcyu** [comment]
> I understand normalization strategies can enrich data in different ways, but when I compared the results of both analysis types on the same data, I got a correlation coefficient of 0.2.

MaxLFQ intensities from two different FragPipe analysis are not comparable because of the normalization and the MaxLFQ algorithm.

>  I am wondering if the normalization of "intensity" is enough, or if from that output I should use the offline normalization I was performing before (this does a log2 of the data, normalizes the means, and then the slopes). I want to be careful to not over normalize my data (Question 2). Any suggestions?

You could try and compare the approaches you mentions and decide.

Best,

Fengchao
